### ISSUE TITLE
where is the loss fucntion

### ISSUE THREAD

**Issue by horizonheart** [Vladkryvoruchko/PSPNet-Keras-tensorflow#48]

**Comment by liminn** [comment]
See     ```
model.compile(optimizer=sgd,
                  loss='categorical_crossentropy',
                  metrics=['accuracy'])
``` in  `layers_builder.py`
